src/Build/BackEnd/Components/SdkResolution/MainNodeSdkResolverService.cs

```diff
@@ -13,6 +13,7 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Task = System.Threading.Tasks.Task;
```

Comment:
Is this needed?

Reply:
Unfortunately yes. When I add reference to the Microsoft.Build.Tasks, there were conflicts with between `Microsoft.Build.Utilities.Task` and `System.Threading.Tasks.Task`.